src/main/resources/application.yml

```diff
@@ -9,6 +9,10 @@ stlcourts:
     tyler:

```

Comment:
clientURL under "spring" needs to be removed everywhere. It's now under "stlcourts". It looks like this was a bad merge